cellbase-app/app/azure/arm/azuredeploy.json

```diff
@@ -0,0 +1,370 @@
+            }
+        },
+        "mongoDBPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "This Data is used to derive unique passwords for several different services. Select a unique and random combination of data, the longer the better, and do not disclose it."
+            }
+        },
+        "mongoDbDumpUrl": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The URL pointing to a tar.gz zip of the a mongo dump"
+            }
+        },
+        "mongNodeCount": {
+            "type": "int",
+            "defaultValue": 3,
+            "allowedValues": [
+                1,
+                3,
+                5,
+                7,
+                9,
+                11
+            ],
+            "metadata": {
+                "description": "Amount of VMs to deploy for the mongodb replica set"
+            }
+        },
+        "mongoNodeSize": {
+            "type": "string",
+            "defaultValue": "Standard_D4s_v3",
+            "metadata": {
+                "description": "The VM size for the mongo nodes"
+            }
+        },
+        "cellbaseContainerImage":{
+            "type": "string",
+            "defaultValue":  "marrobi/cellbase-app"
+        },
+        "customDeploymentSize": {
+            "type": "object",
+            "defaultValue": {},
+            "metadata": {
+                "description": "Define an object for a custom tshirt size of the deployment. WARNING: Intended for experimenting or debugging use with care."
+            }
+        },
+        "deploymentSize": {
+            "type": "int",
+            "allowedValues": [
+                0,
+                1,
+                2
+            ],
+            "defaultValue": 1,
+            "metadata": {
+                "description": "What size of deployment do you want? 0 = For custom deployment. 1 = single mongo node. 2 = mongo replicaset"
+            }
+        },
+        "LogAnalyticsWorkspaceID": {
+            "type": "string",
+            "metadata": {
+                "description": "Log Analytics Workspace ID"
+            }
+        },
+        "logAnalyticsWorkSpaceKey": {
+            "type": "string",
+            "metadata": {
+                "description": "Log Analytics Workspace Key"
+            }
+        },
+        "LogAnalyticsWorkspaceResourceID": {
+            "type": "string",
+            "metadata": {
+                "description": "Log Analytics Workspace Resource ID"
+            }
+        }
+                
+    },
+    "variables": {
+        "mongoTemplateURI":"https://raw.githubusercontent.com/opencb/opencga/052ea4e1be6d708f9dde14ee42ad85914382f8e6/opencga-app/app/scripts/azure/arm/mongodb/azuredeploy.json",

```

Comment:
Nice - good call on pinning to commit. Maybe worth a "_comment" variable on the line above to explain why/how to update that for future users.

Reply:
done